runtime-tests/runtimeTests.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { endpoints } from './endpoints';
+import { PORT, URL_PATH } from './helpers/consts';
+import { request } from './helpers/request';
+import { BlockResponse, ChainConfig, ChainSpec } from './types';
+
+const config = JSON.parse(process.env.__CONFIGURATION as string) as ChainConfig;
+const chain = config.chain as ChainSpec;
+
+const polkadotEndpoints: string[][] = endpoints[chain];
+
+describe('Runtime Tests for blocks', () => {
+	jest.setTimeout(15000);
```

Comment:
What does this timeout do for us?

Reply:
The default timeout in jest is 5000 ms, which sometimes with all the requests can take up to or more than 10000 or more ms. This sets the default timeout to 15000 so the test doesn't fail for a slow response.